server/src/main/java/io/spine/examples/chatspn/server/MessengerContext.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.chatspn.server;
+
+import io.spine.examples.chatspn.server.user.UserAggregate;
+import io.spine.examples.chatspn.server.user.UserProfileRepository;
+import io.spine.server.BoundedContext;
+import io.spine.server.BoundedContextBuilder;
+import io.spine.server.DefaultRepository;
+
+/**
+ * Configures Messenger Bounded Context with repositories.
+ */
+public final class MessengerContext {
+
+    static final String NAME = "Messenger";
+
+    /**
+     * Prevents instantiation of this utility class.
+     */
+    private MessengerContext() {
+    }
+
+    /**
+     * Creates {@code BoundedContextBuilder} for the Messenger context and fills it with
```

Comment:
I don't think that's a "Messenger" bounded context. Usually, a context name is a word used in a real-world domain, and in about 50% of cases, it's plural. E.g. "Tasks", "Chats", etc.

Another thing here is that we don't want to leave orphan-words like "repositories" on the next line. I would move some bits such as "and fills it with" to the next line.